tests/TestCase.php

```diff
@@ -100,6 +100,8 @@ protected function generateCreateParams()
             'meta_presenter'                     => $this->faker->name,
             'meta_endCallbackUrl'                => $this->faker->url,
             'meta_bbb-recording-ready-url'       => $this->faker->url,
+            'bannerText'                         => $this->faker->bannerText,
```

Comment:
Faker doesn't have bannerText. Use something from https://github.com/fzaninotto/Faker

E.g.
```
'bannerText'                         => $this->faker->text,
'bannerColor'                        => $this->faker->hexcolor,
```

Reply:
Sorry, forgot to amend those changes. Should be fixed now.